app/project_type/project_type.py

```diff
@@ -190,17 +194,50 @@ def execute_command_in_project(self, command, extra_environment_vars=None, timeo
         self._logger.debug('Executing command in project: {}', command)
 
         # Redirect output to files instead of using pipes to avoid: https://github.com/box/ClusterRunner/issues/57
-        stdout_file = TemporaryFile()
-        stderr_file = TemporaryFile()
+        output_file = output_file if output_file is not None else TemporaryFile()
         pipe = Popen_with_delayed_expansion(
             command,
             shell=True,
-            stdout=stdout_file,
-            stderr=stderr_file,
+            stdout=output_file,
+            stderr=STDOUT,  # Redirect stderr to stdout, as we do not care to distinguish the two.
```

Comment:
I made a judgment call to just have stdout and stderr be piped to the same file, as we just combine the output from the two streams anyway.

Also, without having just a single console output file, it makes it impossible to implement streaming console output, and we never have the combined output contents in a single file until the very end.